src/server/AuctionHouseServer/AuctionHouse.API/Services/IRegisterUserService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using AuctionHouse.API.DTO;
+
+namespace AuctionHouse.API.Services
+{
+    public interface IRegisterUserService
+    {
+        bool RegisterUser(UserDto userdto);
+        public List<UserDto> GetUserList();
```

Comment:
Retrieving user is not a responsibility of registering service. Instead use UserRepository (maybe behind UserService)